src/utils/test/validAttr.test.js

```diff
@@ -164,245 +164,245 @@ describe('validAttr', () => {
+  //   expect(validAttr('tableValues')).toEqual(true)
+  //   expect(validAttr('targetX')).toEqual(true)
+  //   expect(validAttr('targetY')).toEqual(true)
+  //   expect(validAttr('textAnchor')).toEqual(true)
+  //   expect(validAttr('textDecoration')).toEqual(true)
+  //   expect(validAttr('textRendering')).toEqual(true)
+  //   expect(validAttr('textLength')).toEqual(true)
+  //   expect(validAttr('to')).toEqual(true)
+  //   expect(validAttr('transform')).toEqual(true)
+  //   expect(validAttr('u1')).toEqual(true)
+  //   expect(validAttr('u2')).toEqual(true)
+  //   expect(validAttr('underlinePosition')).toEqual(true)
+  //   expect(validAttr('underlineThickness')).toEqual(true)
+  //   expect(validAttr('unicode')).toEqual(true)
+  //   expect(validAttr('unicodeBidi')).toEqual(true)
+  //   expect(validAttr('unicodeRange')).toEqual(true)
+  //   expect(validAttr('unitsPerEm')).toEqual(true)
+  //   expect(validAttr('vAlphabetic')).toEqual(true)
+  //   expect(validAttr('vHanging')).toEqual(true)
+  //   expect(validAttr('vIdeographic')).toEqual(true)
+  //   expect(validAttr('vMathematical')).toEqual(true)
+  //   expect(validAttr('values')).toEqual(true)
+  //   expect(validAttr('vectorEffect')).toEqual(true)
+  //   expect(validAttr('version')).toEqual(true)
+  //   expect(validAttr('vertAdvY')).toEqual(true)
+  //   expect(validAttr('vertOriginX')).toEqual(true)
+  //   expect(validAttr('vertOriginY')).toEqual(true)
+  //   expect(validAttr('viewBox')).toEqual(true)
+  //   expect(validAttr('viewTarget')).toEqual(true)
+  //   expect(validAttr('visibility')).toEqual(true)
+  //   expect(validAttr('widths')).toEqual(true)
+  //   expect(validAttr('wordSpacing')).toEqual(true)
+  //   expect(validAttr('writingMode')).toEqual(true)
+  //   expect(validAttr('x')).toEqual(true)
+  //   expect(validAttr('xHeight')).toEqual(true)
+  //   expect(validAttr('x1')).toEqual(true)
+  //   expect(validAttr('x2')).toEqual(true)
+  //   expect(validAttr('xChannelSelector')).toEqual(true)
+  //   expect(validAttr('xlinkActuate')).toEqual(true)
+  //   expect(validAttr('xlinkArcrole')).toEqual(true)
+  //   expect(validAttr('xlinkHref')).toEqual(true)
+  //   expect(validAttr('xlinkRole')).toEqual(true)
+  //   expect(validAttr('xlinkShow')).toEqual(true)
+  //   expect(validAttr('xlinkTitle')).toEqual(true)
+  //   expect(validAttr('xlinkType')).toEqual(true)
+  //   expect(validAttr('xmlBase')).toEqual(true)
+  //   expect(validAttr('xmlns')).toEqual(true)
+  //   expect(validAttr('xmlnsXlink')).toEqual(true)
+  //   expect(validAttr('xmlLang')).toEqual(true)
+  //   expect(validAttr('xmlSpace')).toEqual(true)
+  //   expect(validAttr('y')).toEqual(true)
+  //   expect(validAttr('y1')).toEqual(true)
+  //   expect(validAttr('y2')).toEqual(true)
+  //   expect(validAttr('yChannelSelector')).toEqual(true)
+  //   expect(validAttr('z')).toEqual(true)
+  //   expect(validAttr('zoomAndPan')).toEqual(true)
```

Comment:
I was just thinking about this again and wondered if we need to do anything? If we leave all the tests in and someone clones the repo and runs them then they'll pass because at the moment they get `react@15.3.2` from the `devDependencies`. It's only if they go back to an old React 0.14 version that the tests will fail.

@mxstbr, let me know what you'd prefer to have. I can put these failing tests in a conditional like @wmertens suggested (`if (React.version.indexOf('0.14') !== 0) {…}` maybe?) or just leave them as they were and they'll still pass on current versions of React 😄